src/app/Library/CrudPanel/Traits/SaveActions.php

```diff
@@ -417,4 +417,26 @@ public function setupDefaultSaveActions()
 
         $this->addSaveActions($defaultSaveActions);
     }
+
+    /**
+     * Here we check if user should be warned before leaving the page.
+     *
+     * @return bool
+     */
+    public function getWarnBeforeLeave(): bool
```

Comment:
This is a good name, super explicit, but I think `WarnBeforeLeaving` would roll off the tongue more easily (verb instead of noun). I propose we change it to `warnBeforeLeaving` everywhere.